README.md

```diff
@@ -415,8 +409,49 @@ Use `aggregationPipeline` to start a new aggregation. The following operators ar
 
 `aggregate` is currently in an experimental state.
 
+### watch
+
+`fun watch(ignoredFields: List<MongoEntryField<*>> = emptyList(), action: (PayloadChange<Entry>) -> Unit)`
+
+Watch only works if MongoDB is a replica set. Use `ignoredFields` to exclude a set of fields, if any change occurs to these fields it will
+be ignored.
+
+`watch` is currently in an experimental state.
```

Comment:
```suggestion
```
What is experimental about it?

Reply:
Not sure, this line was already there and I didn't want to remove it.